pages/Company-Info-Page.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link";
+import Image from "next/image";
+import type { NextPage } from "next";
+import { useState } from "react";
+
+
+
+const Company:NextPage = () => {
```

Comment:
- `Company` name does not meet the naming conventions for a `page` component. Read the `docs` and rename the page component correctly.